index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@600;700&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+
+      <header>
+        <nav class="nav-bar">
+          <h2 id="header-name">John Doe</h2>    
+          <i id="btn-menu" class="fa-solid fa-bars"></i>
+        </nav>
+      </header>
+
+    <section class="hero">
+
+      <div class="text">
+        <h1>Hey There. <br> I'm Yosef</h1>
```

Comment:
- [ ] Kindly make sure you stick to the design as much as you can, the spacing(margin, padding), typos (fonts) are supposed to be similar to the original template provided in Figma. Kindly make sure you do not have many lines break in this title and match it exactly with the one in Figma.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212111367-97c29d0a-4a48-448e-a296-c86cd060de6b.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/212111551-d1e79f9b-0e96-425b-9438-b464c3a953d5.png)